app/src/main/java/org/mozilla/fenix/addons/AddonsActivity.kt

```diff
@@ -0,0 +1,26 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.fenix.addons
+
+import android.os.Bundle
+import androidx.appcompat.app.AppCompatActivity
+import org.mozilla.fenix.R
+
+/**
+ * An activity to manage add-ons.
+ */
+class AddonsActivity : AppCompatActivity() {
```

Comment:
nit: Fenix usages the [single activity model](https://youtu.be/2k8x8V77CrU), our code should follow the same pattern. Do not worry we can land this as it is and fix it in follow up issue 👍

Reply:
https://github.com/mozilla-mobile/android-components/issues/5374

Reply:
Awesome thanks :)